kernel/src/syscall/rt_sigtimedwait.rs

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::{sync::atomic::Ordering, time::Duration};
+
+use ostd::sync::Waiter;
+
+use super::SyscallReturn;
+use crate::{
+    prelude::*,
+    process::signal::{
+        constants::{SIGKILL, SIGSTOP},
+        sig_action::SigAction,
+        sig_mask::{SigMask, SigSet},
+        with_sigmask_changed, HandlePendingSignal,
+    },
+    time::{timespec_t, wait::ManagedTimeout},
+};
+
+pub fn sys_rt_sigtimedwait(
+    set_ptr: Vaddr,
+    info_ptr: Vaddr,
+    timeout_ptr: Vaddr,
+    sigset_size: usize,
+    ctx: &Context,
+) -> Result<SyscallReturn> {
+    debug!(
+        "set_ptr = 0x{:x}, info_ptr = 0x{:x}, timeout_ptr = 0x{:x}, sigset_size = {}",
+        set_ptr, info_ptr, timeout_ptr, sigset_size
```

Comment:
Using `{:#x}` instead of `0x{:x}` if possible. `{:#x}` will add `0x` prefix automatically.